distributed/tests/test_spill.py

```diff
@@ -24,14 +25,16 @@
 )
 
 
-def psize(*objs: object) -> tuple[int, int]:
+def psize(tmpdir: str, **objs: object) -> tuple[int, int]:
     return (
-        sum(sizeof(o) for o in objs),
-        sum(len(frame) for obj in objs for frame in serialize_bytelist(obj)),
```

Comment:
Instead of reusing the formula for size calculation from `Slow` here, we want to test based on the size of the spilled files.